bser.c

```diff
@@ -48,7 +50,8 @@ static int bser_real(double val, json_dump_callback_t dump, void *data)
   return dump((char*)&val, sizeof(val), data);
 }
 
-bool bunser_string(const char *buf, json_int_t avail, json_int_t *needed,
+/* Common codepath for bytestring and UTF-8 encoded strings. */
+static bool bunser_string(const char *buf, json_int_t avail, json_int_t *needed,
```

Comment:
you may want to rename this or change the type signature to help ensure that you got all references to this name and replaced them with bunser_bytestring